### ISSUE TITLE
DNS issue accessing backend api

### ISSUE THREAD

**Issue by Issam-seghir** [swar8080/obsidian-plugin-update-tracker#53]
I'm use the 1.4.5 version
## My setting
![image](https://user-images.githubusercontent.com/74573675/213124152-a7ea337c-8b23-4b19-ab69-38ae14ff90e6.png)

## The Error

![image](https://user-images.githubusercontent.com/74573675/213123884-daa971e3-22b6-4009-8362-882c9adb7f17.png)

**Comment by swar8080** [comment]
@Issam-seghir  a couple other people reported that requests to the plugin's backend server is blocked on their work computer. It could be that or some other security software blocking the requests

A couple tests to see what the problem:

1. Check if you can access this URL: https://jc5gpa3gs7o2uge6iq5mgjd53q0daomi.lambda-url.us-east-1.on.aws/. The expected response is `Post request expected`

2. Test sending a POST request to the plugin's backend server. https://reqbin.com/adkzdzfn can submit one for you with the same format used by the plugin:
![image](https://user-images.githubusercontent.com/17691679/213566855-3e5f01ca-4dbf-41a1-b01b-200c8005fe0e.png)

Let me know what you see if either of those tests fail

**Comment by Issam-seghir** [comment]
when trying to reach the link
` https://jc5gpa3gs7o2uge6iq5mgjd53q0daomi.lambda-url.us-east-1.on.aws/`

![image](https://user-images.githubusercontent.com/74573675/213644692-9d586df3-1f0f-47e3-a8d9-cd157200152f.png)

sending a POST request test

![image](https://user-images.githubusercontent.com/74573675/213645129-ce98ff9f-dc94-4d35-89b7-1db28c4ad371.png)

**Comment by swar8080** [comment]
@Issam-seghir thanks for checking...odd that you're getting a DNS error like that

Also, realized that POST request uses `reqbin.com`'s servers to send the request instead of your device's network 😅

A better way to test is in your browser's console with this code:
```
fetch('https://jc5gpa3gs7o2uge6iq5mgjd53q0daomi.lambda-url.us-east-1.on.aws/', {
        method: 'POST',
        body: JSON.stringify({
            "currentPluginVersions": [{
                "obsidianPluginId": "dataview",
                "version": "0.0.0"
            }]
        })
    })
    .then(console.log)
    .catch(console.error)
```
<img width="774" alt="image" src="https://user-images.githubusercontent.com/17691679/213731362-0803f35d-9e37-44b5-8ec7-edf24b2c20a2.png">

**Comment by Issam-seghir** [comment]
![image](https://user-images.githubusercontent.com/74573675/213864373-ca86428d-bc10-45eb-a7e0-b2ad2fdd32ae.png)

**Comment by swar8080** [comment]
@Issam-seghir that's strange, i'm not sure why the IP address can't be resolved. All of the DNS stuff is automatically configured by AWS using their [lambda function urls](https://docs.aws.amazon.com/lambda/latest/dg/lambda-urls.html), so i'm going to assume they configured correctly

If you want to learn more about DNS then googling how to resolve the `ERR_NAME_NOT_RESOLVED` error might work. I'm curious what the solution is

**Comment by Issam-seghir** [comment]
I found the solution in this [article](https://www.hostinger.com/tutorials/err_name_not_resolved)
### TLDR;
> The browser stores DNS data to boost page load speed on revisits. However, an outdated entry in the DNS cache could trigger the ERR_NAME_NOT_RESOLVED error message if the website has moved to a different IP address,  so clearing the DNS cache may fix the `ERR_NAME_NOT_RESOLVED` Problem

Clear the host cache on your browser by following these steps :

1. Open Google Chrome and type `chrome://net-internals/#dns` in the browser’s address bar and press Enter.
1. Click `Clear host cache`.

![image](https://user-images.githubusercontent.com/74573675/213940271-2d59f153-51e3-4a5e-832b-2a9a6283a9fe.png)

### Result 
![image](https://user-images.githubusercontent.com/74573675/213940449-eac00625-fae9-4ed6-b4b6-ff0511919185.png)

**Comment by swar8080** [comment]
@Issam-seghir Nice! Glad it's working
